examples/bri-3/src/bri/workgroup/workgroups/api/err.messages.ts

```diff
@@ -0,0 +1,2 @@
+export const WORKGROUP_NOT_FOUND_ERR_MESSAGE = 'Workgroup does not exist.';
+export const BPISUBJECT_NOT_FOUND_ERR_MESSAGE = 'BpiSubject does not exist.';
```

Comment:
This err message already exists in the bpi subject api. Might be better to use the one from there to avoid duplication. We ll introduce a common place for these messages in the future.

Reply:
removed and references original.